fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -101,10 +97,9 @@
 public class LoadManager implements Writable {
     private static final Logger LOG = LogManager.getLogger(LoadManager.class);
 
-    private Map<Long, LoadJob> idToLoadJob = Maps.newConcurrentMap();
-    private Map<Long, Map<String, List<LoadJob>>> dbIdToLabelToLoadJobs = Maps.newConcurrentMap();
-    private LoadJobScheduler loadJobScheduler;
-    private CleanCopyJobScheduler cleanCopyJobScheduler;
+    protected Map<Long, LoadJob> idToLoadJob = Maps.newConcurrentMap();
```

Comment:
for line 100 ~ 102,   if they are only use in cloud mode ?  if so, should put these code into CloudLoadManager.